src/platform/packages/shared/response-ops/alerts-table/hooks/use_bulk_actions.ts

```diff
@@ -283,6 +293,61 @@ export const useBulkUntrackActions = ({
   ]);
 };
 
+export const useBulkRemoveFromCaseActions = ({
+  casesService,
+  casesConfig,
+  refresh,
+  clearSelection,
+  caseId,
+}: UseBulkRemoveFromCaseActionsProps): BulkActionsConfig[] => {
+  const userCasesPermissions = useMemo(() => {
+    return casesService?.helpers.canUseCases(casesConfig?.owner ?? []);
+  }, [casesConfig?.owner, casesService]);
+  const CasesContext = useMemo(() => casesService?.ui.getCasesContext(), [casesService]);
+  const isCasesContextAvailable = Boolean(casesService && CasesContext);
+
+  const onSuccess = useCallback(() => {
+    refresh();
+    clearSelection();
+  }, [clearSelection, refresh]);
+
+  const removeAlertModal = casesService?.hooks.useRemoveAlertFromCaseModal({
+    onSuccess,
+  });
+
+  return useMemo(() => {
+    return isCasesContextAvailable &&
+      removeAlertModal &&
+      userCasesPermissions?.create &&
+      userCasesPermissions?.read &&
+      caseId
+      ? [
+          {
+            label: REMOVE_FROM_CASE,
+            key: 'remove-from-case',
+            'data-test-subj': 'remove-from-case',
+            disableOnQuery: true,
+            disabledLabel: REMOVE_FROM_CASE,
+            onClick: (alerts?: TimelineItem[]) => {
```

Comment:
@elastic/kibana-cases - we should decouple alerts references from timeline references